probe-rs-cli-util/src/rtt.rs

```diff
@@ -96,7 +102,12 @@ pub struct RttChannelConfig {
     pub data_format: DataFormat,
     #[structopt(skip)]
     #[serde(default)]
+    // Control the inclusion of timestamps for DataFormat::String.
     pub show_timestamps: bool,
+    #[structopt(skip)]
+    #[serde(default = "default_include_location")]
```

Comment:
why not directly set the default to true instead of the function? :)

Reply:
The default for bool is false, the only way to set it to true is to call a function. Unless I misunderstand the serde documentation.